hnn_core/tests/test_dipole.py

```diff
@@ -44,6 +44,28 @@ def test_dipole(tmpdir):
                        "average of 2 trials"):
         dipole_avg = average_dipoles([dipole_avg, dipole_read])
 
+    # test postproc
+    hnn_core_root = op.dirname(hnn_core.__file__)
+    params_fname = op.join(hnn_core_root, 'param', 'default.json')
+    params = read_params(params_fname)
+    params_reduced = params.copy()
+    params_reduced.update({'N_pyr_x': 3,
+                           'N_pyr_y': 3,
+                           'tstop': 25,
+                           't_evprox_1': 5,
+                           't_evdist_1': 10,
+                           't_evprox_2': 20,
+                           'N_trials': 2})
+    net = Network(params_reduced)
+    dpls_raw = simulate_dipole(net, postproc=False)
+    dpls = simulate_dipole(net, postproc=True)
+    with pytest.raises(AssertionError):
+        assert_allclose(dpls[0].data['agg'], dpls_raw[0].data['agg'])
```

Comment:
It might be a good idea to explicitly set the relative (`rtol`) and absolute (`atol`) tolerance args for [`assert_allclose()`](https://numpy.org/doc/stable/reference/generated/numpy.testing.assert_allclose.html) so that we can control how much error is acceptable. The defaults are `rtol=1e-07` and `atol=0`.

Reply:
Defaults are usually pretty stringent. Unless there is a specific reason to have more loose checks ...

Reply:
See https://github.com/jonescompneurolab/hnn-core/pull/188#issuecomment-730815655.